cluster/ci/config/prow/openshift/plank.yaml

```diff
@@ -1,34 +1,38 @@
 kind: List
 apiVersion: v1
 items:
-- apiVersion: apps.openshift.io/v1
-  kind: DeploymentConfig
+- apiVersion: extensions/v1beta1
```

Comment:
you should be able to do apps/v1

Reply:
All of the deployments I see on the cluster are `extensions/v1beta1`